vaadin-platform-test/src/test/java/com/vaadin/platform/test/ChromeComponentsIT.java

```diff
@@ -547,6 +548,17 @@ public void contextMenuIsRenderedAndRecievesItemSelectionEvents() {
         assertLog("Context menu Item 0 is clicked");
     }
 
+    @Test
+    public void collaborationAvatarGroupIsRendered() {
+        AvatarGroupElement group = $(AvatarGroupElement.class)
+                .id("collaborationavatargroup");
+        assertElementRendered(group);
+        Assert.assertEquals("foo", group.getAvatarElement(0)
+                .getPropertyString("name"));
+        Assert.assertEquals("bar", group.getAvatarElement(1)
+                .getPropertyString("name"));
```

Comment:
Shouldn't we test that changing an input something appears somewhere indicating that the CE is working and the field is being modified?

Reply:
Discard my comment, I see now the approach